### PR TITLE
[New Site] update banner to announce election

### PR DESCRIPTION
**Pull request by bendhouseart**

### CODE REVIEW

File: docs/overrides/main.html [hunk 1]
```diff
@@ -12,16 +12,22 @@
...
-
+  -->
+    <h2><b>BIDS steering group election</b></h2>
+    This year's election has started. 
+    
+    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
+    
+    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
+    
+    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>
```
Comment (effigies) [inline docs/overrides/main.html:29]:
```suggestion
    <h2><b>BIDS steering group election</b></h2>
    <p>This year's election has started.     
    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
    </p>
```

File: docs/overrides/main.html [hunk 2]
```diff
@@ -12,16 +12,20 @@
...
-<!--
-    <h2>BIDS steering group election</h2>
-
+  -->
+    <h2><b>BIDS steering group election</b></h2>
+    <p>This year's election has started.     
+    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
+    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
+    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
+    </p>
```
Comment (Remi-Gau) [inline docs/overrides/main.html:27]:
```suggestion
    <p>This year's election has started.     
    If you are <a href="https://bids-specification.readthedocs.io/en/latest/appendices/contributors.html">a BIDS contributor</a>,
    please check your BIDS email for an <a href="https://opavote.com" target="_blank">opavote voting link</a> to cast your vote.
    </p>    
    <p>    
    The polls will open on <b>Monday, October 21st</b> and close on <b>Friday, November 1st</b>.
    </p>
```

Comment (Remi-Gau) [reply docs/overrides/main.html]: to avoid very long line

image

**Review by Remi-Gau — APPROVED** [review]
go go go

**Comment by bendhouseart** [comment]
> go go go

As soon as the CI says okay!

**Comment by Remi-Gau** [comment]
> As soon as the CI says okay!

where is the fun in that?! 😉

**Pull request merged**